ui/src/app/pages/workspace/workspace-nav-bar.tsx

```diff
@@ -182,7 +182,7 @@ export const WorkspaceNavBar = fp.flow(
       tabs.push({ name: 'Data Explorer', link: 'data-explorer' });
     }
     if (
-      serverConfigStore.get().config.enableTanagra &&
+        getCdrVersion(workspace, cdrVersionTiersResponse).tanagraEnabled &&
```

Comment:
Failing ui test due to the indentation here, should just need to remove one tab on the front of this line